### DISCUSSION TITLE
Going outsite the domain

### DISCUSSION THREAD

**Discussion by kuolemaaa** [edoardottt/cariddi#165] [Q&A]
I would like to generate the list of all the URLs I can reach starting from a specific host including every URLs that points to another host.

Here an example:
Starting from  `https://subdomain.domain.com/path/`
print all the url in the form `domain.com`, subdomain included, (let's say in a regex form `/.*domain\.com.*/`)
PLUS print also the URLs that point to another host and stop there

**Comment by edoardottt** [comment]
I'm not completely sure what you're trying to do, if I understood correctly the flag `  -intensive` is what you're looking for

**Reply by kuolemaaa** [reply]
Not really. These pages that I'm getting do have some link to external websites. I need to list those links.

**Reply by edoardottt** [reply]
try to make an example, I'm not understanding what you want to achieve
